cli/server.go

```diff
@@ -28,51 +28,101 @@ var (
 	Version string
 )
 
-func cmdServe() *cobra.Command {
-	return &cobra.Command{
-		Use:     "serve",
-		Short:   "Serve gRPC & HTTP service",
-		Long:    heredoc.Doc(`Serve gRPC & HTTP on a port defined in PORT env var.`),
-		Aliases: []string{"server", "start"},
+func serverCmd() *cobra.Command {
```

Comment:
_Currently not able to test these since the config loading is broken._

Reply:
can you once again try with `--config` flag to use the file from current directory. Alternatively you can use the config file created after init command.

Reply:
will test once the current set of review comments are addressed.

Reply:
not sure if you realised but https://github.com/odpf/compass/pull/211#discussion_r1116555739 means that loading config is also broken.